.editorconfig

```diff
@@ -102,12 +102,12 @@ csharp_style_var_when_type_is_apparent = true:suggestion
 
 # Expression-bodied members
 csharp_style_expression_bodied_accessors = true:suggestion
-csharp_style_expression_bodied_constructors = false
+csharp_style_expression_bodied_constructors = false:silent
```

Comment:
What's the difference between `:silent` and not having the line at all?

Reply:
I'm going to revert the `*:silent` changes here.  I don't know why VS added them and I just realized that it can cause errors to show up in the error list when building but then you don't see the analyzer squiggle in the editor which is confusing.